gradle-module-plugin/src/main/kotlin/io/ia/sdk/gradle/modl/task/SignModule.kt

```diff
@@ -194,6 +250,26 @@ open class SignModule @Inject constructor(_providers: ProviderFactory, _objects:
                 throw Exception("Signing key file ${keystoreFile.absolutePath} did not exist!")
             }
         }
+
+        // PKCS#11 HSM (hardware key)-based keystore
+        if (pkcs11Cfg.isPresent) {
```

Comment:
This and the whole `getKeyStore` function should arguably be refactored away in favor of `module-signer`'s existing keystore + key-fetching logic.